sbi/inference/posteriors/base_posterior.py

```diff
@@ -294,3 +214,200 @@ def __setstate__(self, state_dict: Dict):
             state_dict: State to be restored.
         """
         self.__dict__ = state_dict
+
+
+class NeuralPotentialPosterior(NeuralPosterior):
```

Comment:
So this should exactly match what `NeuralPosterior` was before? Double check this.